components/brave_rewards/resources/page/BUILD.gn

```diff
@@ -2,9 +2,12 @@ import("//brave/components/common/typescript.gni")
 
 transpile_web_ui("page") {
   # what is the output bundle called and what is the entry point file
-  entry_points = [
-    ["brave_rewards_page", rebase_path("brave_rewards_page.tsx")]
-  ]
+  entry_points = [ [
+        "brave_rewards_page",
+        rebase_path("brave_rewards_page.tsx"),
+      ] ]
+
+  deps = [ "//brave/vendor/bat-native-ledger/src/bat/ledger/internal/endpoints:error_types_js" ]
```

Comment:
I'm not completely sure about how dependencies are supposed to be laid out, but I think the intention is that other parts of the codebase shouldn't reach into `bat/ledger/internal`. I'm fairly certain that this needs to be in `public`.

Reply:
Put `error_types.mojom` in `vendor/bat-native-ledger/include/bat/ledger/public/interfaces` and renamed it to `ledger_endpoints.mojom`. (Not triggering JavaScript generation here for now.)